ui/pages/confirmations/components/confirm/info/personal-sign/personal-sign.test.tsx

```diff
@@ -2,21 +2,19 @@ import React from 'react';
 import configureMockStore from 'redux-mock-store';
 
 import { renderWithProvider } from '../../../../../../../test/lib/render-helpers';
-import { unapprovedPersonalMsg } from '../../../../../../../test/data/confirmations/personal_sign';
-import PersonalSign from './personalSign';
+import { unapprovedPersonalSignMsg } from '../../../../../../../test/data/confirmations/personal_sign';
+import PersonalSignInfo from './personal-sign';
 
-describe('personalSign', () => {
-  it('renders origin for personal sign request', () => {
+describe('PersonalSignInfo', () => {
```

Comment:
In case the currentConfirmation can be undefined, should we add a test for that?

Reply:
Not really... but singe hook can possible return undefined, TS requires me to handle it